advanced/Scripts/list.sh

```diff
@@ -32,12 +32,18 @@ helpFunc() {
     if [[ "${listType}" == "whitelist" ]]; then
         param="w"
         type="whitelist"
-    elif [[ "${listType}" == "regex_blacklist" && "${wildcard}" == true ]]; then
-        param="-wild"
-        type="wildcard blacklist"
     elif [[ "${listType}" == "regex_blacklist" ]]; then
         param="-regex"
         type="regex blacklist filter"
+    elif [[ "${listType}" == "regex_blacklist" && "${wildcard}" == true ]]; then
+        param="-wild"
+        type="wildcard blacklist"
```

Comment:
The previous if statement will be executed if `listType` is `regex_blacklist` regardless of `wildcard` due to this change (this code will never be executed). The same follows for `wildcard whitelist`.